csv-validator-ui/src/main/scala/uk/gov/nationalarchives/csv/validator/ui/CsvValidatorUi.scala

```diff
@@ -502,6 +514,8 @@ object CsvValidatorUi extends SimpleSwingApplication {
     cbFailOnFirstError.tooltip = "Indicates whether to fail on the first error, or whether to collect all errors!"
     private val cbValidateUtf8 = new CheckBox("Validate CSV for valid UTF-8 characters")
     cbValidateUtf8.selected = true
+    private val tfDisplayLinesLabel = new Label("Maximum number of lines to display:")
```

Comment:
Should this be "maximum number of **errors** to display"? We show success in the same multiline text and we are not truncating success messages, right? 

You are absolutely right in having a colon at the end of the label, however, now that label is odd one out in the settings panel. Can you please add colons to the other labels as well

Reply:
Ah, yes. good point; I'll change it to "errors". Sure, I'll add the colons too.